_runners/vault.py

```diff
@@ -73,7 +73,10 @@ def _get_policies(minion_id, config):
 
   policies = []
   for pattern in policyPatterns:
-    policies.append(pattern.format(**mappings))
+    try:
+      policies.append(pattern.format(**mappings))
+    except KeyError:
+      log.error('Could not resolve policy pattern {0}'.format(pattern))
```

Comment:
I'd suggest making this warning or some lower level. In our environment we'd like to declare a few different policy cases, and gracefully not apply them as necessary. So we wouldn't consider this to be a error level failure, but part of the design.